yb-voyager/cmd/templates/migration_assessment_report.template

```diff
@@ -355,7 +355,7 @@
                 {{ range $idx, $issue := .Issues }}
                     <tbody class="issue-summary-detail-pair"
                         data-category="{{ $issue.Category }}"
-                        data-type="{{ $issue.Type }}"
+                        data-name="{{ $issue.Name }}"
```

Comment:
Are these "data-" fields even being used? I don't see any use 🤔

Reply:
that is used for sorting, in `sortTableBy()` javascript function of the template

Reply:
```
  issuePairs.sort((a, b) => {
                let aValue = a.dataset[field] || "";
                let bValue = b.dataset[field] || "";

                if (aValue < bValue) return newDirection === 'asc' ? -1 : 1;
                if (aValue > bValue) return newDirection === 'asc' ? 1 : -1;
                return 0;
            });
```

Reply:
https://www.linkedin.com/advice/1/how-do-you-use-html5-data-prefix-skills-front-end-development#:~:text=HTML5%20introduced%20a%20new%20feature,styling%2C%20scripting%2C%20and%20validation.